egs2/meld/asr1/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train"
+valid_set="valid"
+test_sets="valid test"
+
+asr_config=conf/tuning/train_asr_hubert_transformer_adam_specaug_meld.yaml
+inference_config=conf/decoder_asr.yaml
+
+./asr.sh \
+    --lang en \
+    --ngpu 1 \
+    --nj 8 \
+    --stop_stage 13 \
+    --inference_config "${inference_config}" \
+    --use_lm false \
+    --nbpe 850 \
+    --token_type bpe\
+    --bpe_train_text "dump/raw/train/text" \
```

Comment:
This is automatically set, we can safely remove this.

Reply:
Thanks, I will remove `--use_lm` and `--bpe_train_text`. I'm not sure if `--nbpe` and `--token_type` are set by default.